doc/development/guide/pullrequests.rst

```diff
@@ -1,74 +1,58 @@
 
-      pylint pennylane/path/to/modified/file.py
+.. image:: view_doc_build.png
+    :width: 400px
+    :align: center
 
-Submitting to the PennyLane repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Ready for review?
+-----------------
 
-When ready, submit your fork as a `pull request <https://help.github.com/articles/about-pull-requests>`_
-to the PennyLane repository, filling out the pull request template. This template is added
-automatically to the comment box when you create a new issue.
+A review-ready pull request (PR) includes the following:
 
-* When describing the pull request, please include as much detail as possible
-  regarding the changes made/new features added/performance improvements. If including any
-  bug fixes, mention the issue numbers associated with the bugs.
+* **All new code is clearly commented and documented.**  See our :doc:`documentation guidelines <documentation>` for more details.
 
-* Once you have submitted the pull request, three things will automatically occur:
+* **All changes must include tests.** If you fix a bug, write a test that would fail before the bugfix. See :doc:`Tests <tests>` for more information.
 
-  - The **test suite** will automatically run on `GitHub Actions <https://github.com/PennyLaneAI/pennylane/actions>`_
-    to ensure that all tests continue to pass.
+* **Code conforms to PEP8 standards.** As mentioned in the section above, all code needs to be formatted via `Black <https://black.readthedocs.io/en/stable/>`_ with a line length of 100.
 
-  - Once the test suite is finished, a **code coverage report** will be generated on
-    `Codecov <https://codecov.io/gh/PennyLaneAI/pennylane>`_. This will calculate the percentage
-    of PennyLane covered by the test suite, to ensure that all new code additions
-    are adequately tested.
+* **Write a changelog entry for all Pull Requests.** The changelog entry should include a link back to the PR. More significant user-facing changes should include a code example. In addition to the changelog entry itself, add your name to the alphabetical contributors' list at the bottom of each release's section.
 
-  - Finally, the **code quality** is calculated by
-    `Codefactor <https://www.codefactor.io/repository/github/pennylaneai/pennylane>`_,
-    to ensure all new code additions adhere to our code quality standards.
+Stuck? You can make a "Work in Progress" PR with ``[WIP]`` in the title or with the "WIP" label.  That way, we can see your work and give feedback early.
 
-Based on these reports, we may ask you to make small changes to your branch before
-merging the pull request into the master branch. Alternatively, you can also
-`grant us permission to make changes to your pull request branch
-<https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/>`_.
+Please note that our review process can take some time and require a bit of back and forth, but hopefully, we all learn something along the way.
```

Comment:
Perhaps better to word this to explain _why_ the review process can take some time? (e.g., provide the positive reasons for this).